src/compare_main.cpp

```diff
@@ -64,7 +65,7 @@ static void show_compare_usage() {
 using SampleIdText = std::string;
 using SampleFpath = std::string;
 
-std::map<SampleIdText, SampleFpath> load_read_index(const std::string &read_index_fpath) {
+std::vector<std::pair<SampleIdText, SampleFpath>> load_read_index(const std::string &read_index_fpath) {
```

Comment:
In trying to be consistent with our use of `boost::filesystem::path` may be better to change this to take a `fs::path` object.

Reply:
I think it is better to address all such cases when solving https://github.com/rmcolq/pandora/issues/68 and https://github.com/rmcolq/pandora/issues/139